api/protobuf/core/api_container_service.proto

```diff
@@ -428,3 +430,21 @@ message FilesArtifactNameAndUuid {
 message ListFilesArtifactNamesAndUuidsResponse {
   repeated FilesArtifactNameAndUuid file_names_and_uuids = 1;
 }
+
+// ==============================================================================================
+//                               Inspect Files Artifact Contents
+// ==============================================================================================
+
+message InspectFilesArtifactContentsRequest {
+  FilesArtifactNameAndUuid file_names_and_uuid = 1;
+}
+
+message InspectFilesArtifactContentsResponse {
+  repeated FileArtifactContentsFileDescription file_descriptions = 1;
+}
+
+message FileArtifactContentsFileDescription {
+  string path = 1;
```

Comment:
what is the path relative to? the path of the file in the tar ball we create?

Reply:
Yes. Added a comment for clarification